api/api.csproj

```diff
@@ -22,4 +22,40 @@
     <DotNetCliToolReference Include="Microsoft.EntityFrameworkCore.Tools.DotNet" Version="2.0.0" />
     <DotNetCliToolReference Include="Microsoft.DotNet.Watcher.Tools" Version="2.0.0" />
   </ItemGroup>
+  
+  
+  <Target Name="DebugRunWebpack" BeforeTargets="Build" Condition=" '$(Configuration)' == 'Debug' ">
+	<Message Importance="high" Text="Installing any non-peer missing dependencies.." />
+    <Exec Command="npm install" />
```

Comment:
This runs `npm install` on every build which is redundant to the existing `npm install` [postinstall](https://github.com/bradymholt/aspnet-core-react-template/blob/master/package.json#L7) scripts.  Also, `npm install` is not terribly fast so this slows build down.  Using npm script flow makes this redundant but I understand that using npm command line might be not standard/preferred on Windows.  What is the typical flow for running `npm install` in Windiows/Visual Studio?

Reply:
The only check for this is to ensure resources exist prior to launch, but I have to agree entirely! I debated removing it for speed, but wasn't sure what would be the community accepted standard so to speak. The readme covers installation so it does seem unnecessary! Can get that removed